File: tcc-college/Cult/cult/lib/Models/city.dart
class City {
  int id;
  String name;
  String stateCode;

  City(int id, String name, String stateCode) {
    this.id = id;
    this.name = name;
    this.stateCode = stateCode;
  }

  City.fromJson(Map json)
      : id = json['id'],
        name = json['name'],
        stateCode = json['state_code'];
}

File: tcc-college/Cult/api/src/models/City.js
const mongoose = require("../../config/database");

const citySchema = mongoose.Schema({
  name: {
    type: String,
  },
  state_id: {
    type: Number,
  },
  state_code: {
    type: String,
  },
  country_id: {
    type: Number,
  },
  country_code: {
    type: String,
  },
  image: {
    type: String,
  },
  featured: {
    type: Boolean,
    default: false,
  },
  latitude: {
    type: Number,
  },
  longitude: {
    type: Number,
  },
});

module.exports = mongoose.model("City", citySchema);
